static/src/js/zustand/slices/__tests__/createProjectSlice.test.ts

```diff
@@ -395,8 +394,6 @@ describe('createProjectSlice', () => {
           }
         })
 
-        expect(actions.setDataQualitySummaries).toHaveBeenCalledTimes(0)
```

Comment:
Each of these removed lines was ensuring this call didn't happen when it shouldn't happen. You can set `setDataQualitySummaries` to a mock function, then keep the assertion that the function wasn't called